.clang-format

```diff
@@ -14,3 +14,8 @@ SortIncludes: true
 BreakBeforeTernaryOperators: true
 BinPackArguments: false
 BinPackParameters: false
+AllowShortCompoundRequirementOnASingleLine: true
+IndentRequiresClause: true
+RequiresClausePosition: OwnLine
+RequiresExpressionIndentation: OuterScope
+AllowShortLambdasOnASingleLine: All
```

Comment:
I had to update clang-format now because the version we used didn't really understand `requires` clauses and made a huge mess of functions using it.